src/plugins/search/component.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-unused-vars */
 import { search } from './search.js';
```

Comment:
Let's remove unused var

Reply:
ESLint's `no-unused-vars` is configured in `eslint.config.js`. The comment is auto-removed when running `lint:fix` because it is unnecessary. The rule is still enforced.

Reply:
Oh ok great!